packages/common/src/ui/components/popover/PaperPopover/PaperPopover.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { FC } from 'react';
+import { HoverPopover } from '../HoverPopover';
+import { useTranslation, LocaleKey } from '../../../../intl';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import Paper, { PaperProps } from '@mui/material/Paper';
+
+interface PaperPopoverProps {
+  Content: React.ReactElement<any, any>;
+  paperProps?: PaperProps;
+  width?: number;
+  height?: number;
+  placement?: 'top' | 'bottom' | 'left' | 'right';
+}
+
+export const PaperPopover: FC<PaperPopoverProps> = ({
+  children,
+  Content,
+  paperProps,
+  width = 250,
+  height = 125,
+  placement = 'left',
+}) => {
+  return (
+    <HoverPopover
+      placement={placement}
+      Content={
+        <Paper
+          sx={{
+            width,
+            height,
+            boxShadow: theme => theme.shadows[7],
```

Comment:
would you mind adding 
```
            borderRadius: '16px',
```
as well? Manu's designs have the exaggerated round corners, so I think worth putting in at the `Paper` level